core/src/main/scala/org/apache/spark/deploy/SparkHadoopUtil.scala

```diff
@@ -450,13 +450,22 @@ private[spark] object SparkHadoopUtil {
           hadoopConf.set("fs.s3a.session.token", sessionToken)
         }
       }
+      loadHiveConfFile(conf, hadoopConf)
       appendSparkHadoopConfigs(conf, hadoopConf)
       appendSparkHiveConfigs(conf, hadoopConf)
       val bufferSize = conf.get(BUFFER_SIZE).toString
       hadoopConf.set("io.file.buffer.size", bufferSize)
```

Comment:
Hi, @MaxGekk . According to your email, is this the only property affected?

Reply:
@dongjoon-hyun Yes.